Rework README.md

rshym opened the pull request:
Provide info regarding the preparation of the host, building and flashing.

rshym commented:
@svlad-90, @Mishytca, @lorc, could you please take a look?

Text is based on the https://kb.epam.com/pages/viewpage.action?spaceKey=EPMDAEPR&title=102124+Instruction+for+building+and+flash+the+KF+Cluster+setup

The main idea behind this PR was to get small instructions that would cover all the steps required to build and flash the product.

Mishytca commented on README.md:
```diff
@@ -1,178 +1,243 @@
+https://gitbud.epam.com/epmp-aos/kf_prebuilt_graphics
 
-Moulin will generate `build.ninja` file. After that - run `ninja` to build the images. This will take some time and disk space, as it will built 3 separate Yocto images.
+Download nfs.tar.bz2 and tftp.tar.bz2 from this repo
```
Hi @rshym .
The repository https://gitbud.epam.com/epmp-aos/kf_nfs_and_tftp_data)had issues only with the `nfs` archive.
Now the repository contains the following archives:

- nfs.tar.bz2
- tftp.tar.gz

Please update the archive names in the README to match those in the above repository.

rshym replied:
Thanks for catching this! Yes, you are right, but right now, I'm preparing PR for that repo, adding tftp.tar.bz2 with a modified path and the name of the .dtb.

I will add the corresponding link.

rshym replied:
Here is PR that adds tftp.tar.bz2 file to the `kf_nfs_and_tftp_data` 
https://gitbud.epam.com/epmp-aos/kf_nfs_and_tftp_data/-/merge_requests/1

Mishytca replied:
Hi @svlad-90, please merge this commit as I don't have permission in this repository.

rshym commented:
Force-pushed due to fixed formatting. Empty line is added:
```
 - Hardware/Software Flow Control - No
+
 Press Enter and select "Save setup as dfl".
 ```

rshym commented:
- fixed command line option for rcar_flash
- added installation of the pyserial and pyftdi, required by rcar_flash
- added info regarding adding the user to the dialout group for the using minicom without sudo
- 'minicom' section is renamed to 'serial port tools'

svlad-90 commented on README.md:
````diff
@@ -1,178 +1,251 @@
+```
+In the u-boot, start BSP
+```
+==> run bootcmd_bootcmd_bsp
````
Hi @rshym, You only have the 'bootcmd_bsp' command. I guess there is a typo here. Please fix it.

rshym replied:
Thanks for the rigorous reading! Fixed.

svlad-90 commented:
Reviewed-by: Vladyslav Goncharuk <vladyslav_goncharuk@epam.com>

rshym commented:
Rb is added.
Ready to be merged.
